### PR TITLE
Enhance Romanian Translations in ro.json for Mammotion Component

### PR DESCRIPTION
**Pull request by virusczz**
### **User description**
update to 0.1.21

___

### **Description**
- Enhanced the Romanian translations for the Mammotion component by adding new configuration options and entity attributes.
- Introduced emergency nudge actions for better user control.
- Updated cutting mode options to include more states for improved functionality.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>ro.json</strong><dd><code>Enhance Romanian Translations for Mammotion Component</code>&nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

custom_components/mammotion/translations/ro.json
<li>Added new configuration options for the Mammotion component.<br> <li> Introduced new entity attributes for enhanced functionality.<br> <li> Expanded button actions with emergency nudge options.<br> <li> Updated cutting mode options with additional states.<br>

</details>

  </td>
  <td><a href="https://github.com/mikey0000/Mammotion-HA/pull/110/files#diff-3f58c6ec291852730bb27ebf46ab5f03c58b85852c9b09fe77e60293af93416d">+38/-0</a>&nbsp; &nbsp; </td>
</tr>
</table></td></tr></tr></tbody></table>

### CODE REVIEW

**Comment by mikey0000** [comment]
@virusczz could you update the rest that I have in this branch? Then I'll merge :)

**Comment by virusczz** [comment]
done

**Review by mikey0000 — APPROVED** [review]

**Comment by virusczz** [comment]
done

File: custom_components/mammotion/translations/ro.json [hunk 1]
```diff
@@ -4,15 +4,17 @@
...
       "not_supported": "Dispozitivul nu este suportat",
-      "no_device_name": "Numele dispozitivului este necesar pentru a continua",
       "reconfigure_successful": "Reconfigurare reușită"
     },
     "flow_title": "Configurați mașina de tuns iarba Mammotion",
     "step": {
       "bluetooth_confirm": {
-        "description": "Configurație{name}"
+        "description": "Configurație{name}",
+        "stay_connected_bluetooth": "Mențineți bluetooth-ul conectat"
```
Comment (mikey0000) [inline custom_components/mammotion/translations/ro.json:17]:
```suggestion
        "data": {
            "stay_connected_bluetooth": "Mențineți bluetooth-ul conectat"
        }
```

**Pull request merged**